### ISSUE TITLE
Include is_retracted and is_paratext in oa2df?

### ISSUE THREAD

**Issue by adam3smith** [massimoaria/openalexR#27]
Thanks for this package -- very useful. I'd like to filter out irrelevant results from a sample that I'm constructing using oa_random (so can't use the API filters) and it'd be very useful to have access to `is_retracted` and `is_paratext` -- happy to create a PR (this seems pretty straightforward) but wanted to make sure these aren't committed for a reason.

**Comment by trangdata** [comment]
Thanks for raising this point @adam3smith! Could you explain a little more what you're trying to do? Are you using oa_random multiple times to get a set of random works? PRs are most welcomed, but I don't think you want to modify `oa2df`...

**Comment by trangdata** [comment]
I see your point now! 💯  And yes, then adding these two columns to works2df would make sense! Please submit a PR! 🙏🏽 

Alternatively, if you want more control of the output, I would also use `output = "list"` in your oa_random calls and then rbind() %>% as_tibble() them together at the end. You would get a tibble with some nested lists (not as nice as what works2df gives you) but maybe we could modify works2df to take multiple output lists as well.

**Comment by trangdata** [comment]
@adam3smith OpenAlex can now return [more than one](
https://docs.openalex.org/how-to-use-the-api/get-lists-of-entities/sample-entity-lists) random work/author/etc. per API call. So you can query
https://api.openalex.org/works?filter=has_doi:true,type:journal-article,is_paratext:false,has_references:false&sample=1000

And with the added `sample` argument implemented in #82:

``` r
adam_ex <- oa_fetch(
  "works",
  has_doi = TRUE,
  type = "journal-article",
  is_paratext = FALSE,
  has_references = FALSE,
  sample = 1000,
  verbose = TRUE
)
```

**Comment by adam3smith** [comment]
ooh that's nifty -- that'd have made things much faster for me (and saved their poor API some stress). Thanks for letting me know (also, now I know why your name on social media seemed familiar ;)
